client/src/EditProperty.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import axios from 'axios';
+
+export default class EditProperty extends React.Component {
+
+
+    render(){
+        const {property} = this.props;
+        
+        return (<div>
+            <form>
+                <div className="mb-3">
```

Comment:
This could be refactored since there is code duplication. The only thing that changes is the id.